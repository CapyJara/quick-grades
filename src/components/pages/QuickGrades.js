import React from 'react';
import Main from '../../containers/Main';
import styles from '../app.css';

const QuickGrades = () => {
  return (
    <section className={styles.app}>
      <h1>Slightly Quickr Grading</h1>
      <Main />
    </section>
  );
};

export default QuickGrades;
