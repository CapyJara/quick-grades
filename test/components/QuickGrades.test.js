import React from 'react';
import { shallow } from 'enzyme';
import QuickGrades from '../../src/components/pages/QuickGrades';

describe('QuickGrades component', () => {
  it('renders QuickGrades', () => {
    const wrapper = shallow(<QuickGrades />);
    expect(wrapper).toMatchSnapshot();
  });
});
