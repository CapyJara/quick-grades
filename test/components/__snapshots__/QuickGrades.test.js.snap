// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QuickGrades component renders QuickGrades 1`] = `
<section
  className="app"
>
  <h1>
    Slightly Quickr Grading
  </h1>
  <Main />
</section>
`;
